server/routes/api/user.js

```diff
@@ -7,27 +7,31 @@ const router = express.Router();
 router.get('/', async (req, res) => {
     const { upn } = req.params;
 
-    const { rows } = await db.query(`
+    const { rows } = await db.query(
+        `
```

Comment:
If you break line for opening `, please break line for the closing one (on line 17) as well, the same way you did on lines 72-75
repeats itself on lines: 28-32, 49-51, 136-138, 148-169

Reply:
:+1: